### ISSUE TITLE
No Teleportation to map

### ISSUE THREAD

**Issue by ItzPike** [SkyWars/SkyWars#251]
#### Issue / Bug / Crash:

Server Version:

1.10

When did this issue occur?:

7/25/17

What did you expect to happen?:

The players queued will be teleported to the map that is on the server files. where skywars starts.

What happened?:

An error got outputted so the players were never teleported and stayed in the lobby.  The game still began and the last person alive wins.

Output url from `/sw report`:

https://gist.github.com/anonymous/cc759d21b9c18bae5497336c0b5b1aed

Any errors in console?:

https://pastebin.com/R5b63jti

#### Feature Request:

What would you like to be added?:

Maybe like a system where it tells you what the problem is and tells you a solution.

**Comment by ItzPike** [comment]
Edit: It won't load the pap I provided in.

**Comment by daboross** [comment]
Definitely an error somewhere in the codebase, I'll look into this.

If you have the time, could you check and see if the message `Updating arena blocks cache for arena` appears anywhere in the log before this error occurred? It seems like there's an error happening where this _isn't_ occurring, and I just want to make sure that diagnostic is correct.

**Comment by ItzPike** [comment]
No, I do not see Updating arena blocks cache for arena. But along with that when the game is finished this error also outputs:
https://pastebin.com/QQC8tCnz

**Comment by ItzPike** [comment]
Any updates?

**Comment by Shinare-Seaweed** [comment]
I have same problem, except that the plugin's default maps work. Only the maps I added give that error.
Server version is 1.7.10. Do you offer any support for old versions?

**Comment by daboross** [comment]
I've not had much time to debug this issue, but if it's broken it should break equally on all server versions. That is to say, I don't currently offer much support for any versions, but there's at least as much support for 1.7.10 as for any other version, if not more.

It definitely seems a bit strange that it would break for only custom maps and not pre-provided ones, but that's useful information.

@Shinare-Seaweed is there any possibility you could zip up your arena's `.blocks` and `.yml` files in `plugins/SkyWars/arenas/` and upload them here? Not a necessity, but it would be helpful to have some files which for sure reproduce this.

If you don't want to make the arenas public, which is understandable, emailing to `daboross@daboross.net` would also work.

@ItzPike My apologies for not getting back to this issue earlier.

**Comment by ItzPike** [comment]
Yeah, I forgot to add on that the default maps work.
But yeah, it's fine that you didn't get back to me further before.

**Comment by Shinare-Seaweed** [comment]
Sure I can upload the map here, however you might not be able to run it, since it was created on modded server with a lot of modded blocks. But I don't think this is the issue, since others have experienced it as well.
[map-mordor2.zip](https://github.com/SkyWars/SkyWars/files/1775532/map-mordor2.zip)
